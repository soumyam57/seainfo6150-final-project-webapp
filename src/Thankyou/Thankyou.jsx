import React from 'react';
import styles from "./Thankyou.module.css";

const Thankyou = () => {
    return (
        <div className={styles.grid}>
            <div className={styles.thankyou}>
                <h3>Your order is confirmed</h3><br />
                <h1>Thank you for shopping with me. Enjoy your day :-) </h1>
            </div>
        </div>
    )
}

export default Thankyou;