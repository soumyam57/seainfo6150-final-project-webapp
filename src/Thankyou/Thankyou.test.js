import React from "react";
import { render } from "@testing-library/react";
import Thankyou from "./Thankyou.jsx";

it("renders the ThankPage component correctly", () => {
  const { container } = render(<Thankyou />);
  expect(container).toMatchSnapshot();
});