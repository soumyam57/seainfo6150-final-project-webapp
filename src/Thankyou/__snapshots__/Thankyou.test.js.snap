// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the ThankPage component correctly 1`] = `
<div>
  <div
    class="grid"
  >
    <div
      class="thankyou"
    >
      <h3>
        Your order is confirmed
      </h3>
      <br />
      <h1>
        Thank you for shopping with me. Enjoy your day :-) 
      </h1>
    </div>
  </div>
</div>
`;
